src/core_write.cpp

```diff
@@ -174,7 +174,8 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
             in.pushKV("vout", (int64_t)txin.prevout.n);
             UniValue o(UniValue::VOBJ);
             o.pushKV("asm", ScriptToAsmStr(txin.scriptSig, true));
-            o.pushKV("hex", HexStr(txin.scriptSig.begin(), txin.scriptSig.end()));
```

Comment:
Removing keys can break existing software. Not sure if this needs deprecation a flag.

Reply:
I only see two consumer (bitcoin-tx.cpp and rest.cpp) and both are not changing the default which is true. So there is no functional change.